src/Request/Request/GameEngine/Event/InputEvent.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace MaxBeckers\AmazonAlexa\Request\Request\GameEngine\Event;
+
+/**
+ * @author Fabian Graßl <fabian.grassl@db-n.com>
+ */
+class InputEvent
+{
+    const ACTION_DOWN = 'down';
+    const ACTION_UP   = 'up';
+
+    /**
+     * @var string
+     */
+    public $gadgetId;
+
+    /**
+     * @var \DateTime
+     */
+    public $timestamp;
+
+    /**
+     * @var string
+     */
+    public $action;
+
+    /**
+     * @var string
+     */
+    public $color;
+
+    /**
+     * @var string
+     */
+    public $feature;
+
+    /**
+     * @param array $amazonRequest
+     *
+     * @throws \Exception
+     *
+     * @return InputEvent
+     */
+    public static function fromAmazonRequest(array $amazonRequest): self
+    {
+        $event = new self();
+
+        $event->gadgetId   = $amazonRequest['gadgetId'];
+        $event->timestamp  = new \DateTime($amazonRequest['timestamp']);
```

Comment:
Do we need here a solution like https://github.com/maxbeckers/amazon-alexa-php/blob/master/src/Request/Request/Standard/StandardRequest.php#L33 ? Maybe the dev console here also works not the expected way.

Reply:
I tried it with the dev console. It returns the right timestamps. Could it be that this issue is resolved?

Reply:
Yes may be. Or in this case it was still working.